source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/skin.html

```
@@ -112,6 +112,35 @@
 </div>
 <div id="caViewportBlocker" class="ca_viewportBlocker ca_hide" aria-hidden="true"></div>
+
+<? if (($GLOBALS['caSettings']['dev'] ?? "no") == "yes"):?>
+<!-- Dev-mode diff overlay: covers .ca_display_area + the menu while the
+     sidebar stays anchored on the right. Populated lazily by
+     caShowTemplateDiff (diff.js, only loaded in dev mode). -->
+<div id='caDiffView' class='ca_hide' role='dialog' aria-modal='true' aria-label='<?tr("Template diff");?>'>
+	<!-- Standalone close — deliberately NOT class='popUpClose' so the body
+	     delegate that calls closeSidebar() doesn't fire alongside the
+	     diff-only dismissal. -->
+	<span class='caButton caDiffClose' role='button' tabindex='0' onclick='caHideTemplateDiff();'><?tr("CLOSE");?></span>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)